test/products/get-one-product-children-details.test.ts

```diff
@@ -49,7 +49,7 @@ describe('v1: Products: can get details of product children', () => {
             200,
             {
               count: 1,
-              results: [{ id: '12345' }]
+              results: [{ id: '12345' }, { id: '456789' }]
```

Comment:
So technically, the count is 2 here, right?

Reply:
Good point - Will remove that line